apps/generic-issuance-client/src/helpers/useFetchPipeline.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  GenericIssuanceGetPipelineResponse,
+  requestGenericIssuanceGetPipeline
+} from "@pcd/passport-interface";
+import { useEffect, useState } from "react";
+import { ZUPASS_SERVER_URL } from "../constants";
+import { useJWT } from "./userHooks";
+
+export function useFetchPipeline(
+  id: string | undefined
+): GenericIssuanceGetPipelineResponse | undefined {
+  const jwt = useJWT();
+  const [result, setResult] = useState<
+    GenericIssuanceGetPipelineResponse | undefined
+  >();
+
+  useEffect(() => {
+    if (id === undefined) {
+      return;
+    }
+
+    requestGenericIssuanceGetPipeline(
+      ZUPASS_SERVER_URL,
+      id ?? "",
+      jwt ?? ""
```

Comment:
similar to id, it should probably just early return if JWT is undefined/null rather than making a request we know will fail